### PR TITLE
Add Tag component

### PR DESCRIPTION
**Pull request by mfrachet**
Add a Tag component

Example: https://design-system-git-add-tag-strapijs.vercel.app/?path=/story/tag--base

## API

```jsx
 <Tag icon={<HelpIcon aria-hidden={true} />}>Hello world</Tag>
```

## Voice over

Using the Tag component using Voiceover

### CODE REVIEW

File: packages/strapi-design-system/src/Tag/Tag.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Text } from '../Text';
+import { Box } from '../Box';
+import { Row } from '../Row';
+
+const TagWrapper = styled(Box)`
+  border: 1px solid ${({ theme }) => theme.colors.primary200};
+  border-radius: ${({ theme }) => theme.borderRadius};
```
Comment (soupette) [inline packages/strapi-design-system/src/Tag/Tag.js:10]: I am being picky here but what do you think about creating a utility to retrieve the borderRadius that we could use in all our component? This is almost always the same code

```
const getBorderRadius = ({ theme }) => theme.borderRadius;


const Compo = styled(Box)`
  border-radius: ${getBorderRadius};
`
```

Comment (mfrachet) [reply packages/strapi-design-system/src/Tag/Tag.js:10]: I understand the intent of avoiding duplication. But adding more code in other places reduces visibility to me. I can make such an utility (wyt @HichamELBSI ?) but does it worth it?

Comment (HichamELBSI) [reply packages/strapi-design-system/src/Tag/Tag.js:10]: A helper for one line, not sure how useful it is. Can you explain the advantage of having this, please?

**Review by HichamELBSI — APPROVED** [review]
LGTM good job

File: packages/strapi-design-system/src/Tag/Tag.js (hunk 1 above)
Comment (soupette) [reply packages/strapi-design-system/src/Tag/Tag.js:10]: I let you decide if you think it's worthing it

**Comment by maevalienard** [comment]
Why didn't you chose to pick the cross icon instead of the information one? 😂

**Comment by mfrachet** [comment]
It's not in strapi/icons I think 😓 (that's the only reasons tbh)

**Comment by maevalienard** [comment]
Except that point, LGTM :)

**Pull request merged**